index.bs

```diff
@@ -1264,7 +1264,7 @@ implementation of this isolation that can be implemented by all browsers.
 
   1. Return |result| and run the remaining steps [=in parallel=].
 
-  1. If |scriptNameList| is undefined, then:
+  1. If |scriptNameList| is null, then:
```

Comment:
I asked Dom about this, and I think we should update the IDL to mark scriptNameList as optional instead of nullable, and then leave this check as-is.